### ISSUE TITLE
Nuget Pack is dropping the last digit of version numbers if its zero

### ISSUE THREAD

**Issue by StingyJack** [NuGet/Home#5225]
## Details about Problem
nuget.exe
v3.5

running the command line below does produce output, but its dropping 0 from the last part of the version number

```
nuget pack .\Common.csproj -Properties "Configuration=Debug;Platform=AnyCPU"  -verbosity  detailed -Symbols -Version "7.5.1.0" -Output "c:\temp\nuget packages"

...
Successfully created package 'c:\temp\nuget packages\Common.7.5.1.symbols.nupkg'.
```
if I change the version parameter to "7.5.1.0-butt", it uses the value specified and honors the -butt

I'm trying to repackage some assemblies that are already built and released for in house reuse, and cant make this accept the version but without the butt. Is there any way to make nuget just use the value I gave it?

**Comment by emgarten** [comment]
The looks related to https://github.com/NuGet/Home/issues/3050

**Comment by StingyJack** [comment]
I did see that, but Its not normalizing it - it's removing significant digits or has a rounding/truncation error or something.

Due to pre-existing contractual concerns and other decisions made years ago, our versioning effectively uses the last three positions as semver. I'm not sure when this started happening as I think we have other similarly numbered packages (ex 7.5.0.0 or the like), but I'll have to check when I get back into the office.

**Comment by mishra14** [comment]
@StingyJack Thanks for the comment. We consider this as a normalization issue and should be fixed as part of #3050. If you want to bring attention to this, please comment on that issue.

**Comment by StingyJack** [comment]
@mishra14 - I'm on that one too, and I'll accept this as a dupe. 

However I dont know of a definition for  "normalizing" that includes destroying or discarding data that isn't part of.  If you normalized records in a rdbms, it would be a de-duplication effort not a removal effort. 

"Normalizing" in this case would mean expanding the shorter version numbers to match the longer ones.
